src/python/zquantum/core/estimator.py

```diff
@@ -38,6 +72,13 @@ def get_estimated_expectation_values(
         self._log_ignore_parameter(estimator_name, "epsilon", epsilon)
         self._log_ignore_parameter(estimator_name, "delta", delta)
 
+        frame_operators = []
```

Comment:
So this is "Basic Estimator using basic grouping technique" right?
Do we plan to use different estimators for doing different grouping approaches?

Reply:
It's measuring each term independently (i.e. no grouping), although in zmachine we called this the "individual" grouping strategy.

I am thinking that grouping (and other Hamiltonian decompositions such as orbital frames) should happen outside of the estimator classes because some decompositions that can be used with more than one estimator. Also because we've found that it can be useful to have several tasks to manipulate and analyze a decomposition before running an optimization.